osc-server/src/main/java/org/osc/core/broker/service/vc/DeleteVirtualizationConnectorService.java

```diff
@@ -18,40 +18,33 @@
 
 import javax.persistence.EntityManager;
 
-import org.osc.core.broker.job.Job;
-import org.osc.core.broker.job.JobEngine;
-import org.osc.core.broker.job.TaskGraph;
-import org.osc.core.broker.job.lock.LockObjectReference;
 import org.osc.core.broker.model.entities.virtualization.VirtualizationConnector;
 import org.osc.core.broker.service.ServiceDispatcher;
 import org.osc.core.broker.service.api.vc.DeleteVirtualizationConnectorServiceApi;
 import org.osc.core.broker.service.exceptions.VmidcBrokerValidationException;
 import org.osc.core.broker.service.persistence.OSCEntityManager;
+import org.osc.core.broker.service.persistence.SslCertificateAttrEntityMgr;
 import org.osc.core.broker.service.persistence.VirtualizationConnectorEntityMgr;
 import org.osc.core.broker.service.request.BaseIdRequest;
-import org.osc.core.broker.service.response.BaseJobResponse;
-import org.osc.core.broker.service.tasks.conformance.virtualizationconnector.VCDeleteMetaTask;
-import org.slf4j.LoggerFactory;
+import org.osc.core.broker.service.response.EmptySuccessResponse;
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-import org.slf4j.Logger;
 
 @Component
-public class DeleteVirtualizationConnectorService extends ServiceDispatcher<BaseIdRequest, BaseJobResponse>
+public class DeleteVirtualizationConnectorService extends ServiceDispatcher<BaseIdRequest, EmptySuccessResponse>
     implements DeleteVirtualizationConnectorServiceApi {
 
-    private static final Logger log = LoggerFactory.getLogger(DeleteVirtualizationConnectorService.class);
-
-    @Reference
-    VCDeleteMetaTask deleteMetaTask;
-
     @Override
-    public BaseJobResponse exec(BaseIdRequest request, EntityManager em) throws Exception {
+    public EmptySuccessResponse exec(BaseIdRequest request, EntityManager em) throws Exception {
         validate(em, request);
 
         OSCEntityManager<VirtualizationConnector> vcEntityMgr = new OSCEntityManager<>(VirtualizationConnector.class, em, this.txBroadcastUtil);
         VirtualizationConnector vc = vcEntityMgr.findByPrimaryKey(request.getId());
-        return new BaseJobResponse(startJob(vc));
+
+        SslCertificateAttrEntityMgr sslCertificateAttrEntityMgr = new SslCertificateAttrEntityMgr(em, this.txBroadcastUtil);
```

Comment:
we should try write lock the vc before we delete. Otherwise if the VC is being actively used somewhere and we delete the VC underneath...its going to result in errors